package.json

```diff
@@ -27,7 +27,7 @@
     "@types/express": "^4.0.33",
     "@types/mime": "0.0.29",
     "@types/mz": "0.0.29",
-    "@types/node": "^4.0.30",
+    "@types/node": "=4.0.30",
```

Comment:
Installing locally throws various typing errors with `error TS2304: Cannot find name 'console'.` in a large number of files. It seems that version `4.2.0` of the typings are breaking installation. Therefore I pinned this version to the last version before `4.2.0`